apps/admin-portal/src/components/applications/advanced-configurations.tsx

```diff
@@ -0,0 +1,182 @@
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Field, Forms } from "@wso2is/forms";
+import React, { FunctionComponent } from "react";
+import { Button, Card, Grid, Header } from "semantic-ui-react";
+import { updateAdvanceConfigurations } from "../../api/application";
+import { AdvancedConfigurationsInterface } from "../../models";
+
+/**
+ *  Advance Configurations for the Application.
+ *
+ * @param props Advance configuration model.
+ */
+interface AdvanceConfigurationExtentedInterface extends AdvancedConfigurationsInterface {
+    appId?: string;
+}
+
+export const AdvanceConfigurations: FunctionComponent<AdvanceConfigurationExtentedInterface> = (props): JSX.Element => {
+
+    const {
+        saas,
+        discoverableByEndUsers,
+        certificate,
+        skipConsent, // TODO  add consent for logout
+        returnAuthenticatedIdpList,
+        enableAuthorization,
+        appId
+    } = props;
+
+    const findSelectedConfiguration = (): string[] => {
+        const selectedValues = []
+        if (saas) {
+            selectedValues.push("saas");
+        }
+        if (discoverableByEndUsers) {
+            selectedValues.push("discoverableByEndUsers");
+        }
+        if (skipConsent) {
+            selectedValues.push("skipConsent");
+        }
+        if (returnAuthenticatedIdpList) {
+            selectedValues.push("returnAuthenticatedIdpList");
+        }
+        if (enableAuthorization) {
+            selectedValues.push("enableAuthorization");
+        }
+        return selectedValues;
+    };
+
+    const handleSubmit = (values) => {
+        updateAdvanceConfigurations(appId, updateConfiguration(values));
+    };
+
+    const updateConfiguration = (values) => {
+
+        return {
+            advancedConfigurations: {
+                certificate: {
+                    type: values.get("type"),
+                    value: values.get("value"),
+                },
+                discoverableByEndUsers:
+                    (values.get("advanceConfiguration").includes("discoverableByEndUsers") ? true : false),
+                enableAuthorization:
+                    (values.get("advanceConfiguration").includes("enableAuthorization") ? true : false),
+                returnAuthenticatedIdpList:
+                    (values.get("advanceConfiguration").includes("returnAuthenticatedIdpList") ? true : false),
+                saas: (values.get("advanceConfiguration").includes("saas") ? true : false),
+                skipConsent: (values.get("advanceConfiguration").includes("skipConsent") ? true : false)
+            }
+        };
+    }
+
+    return (
+        <>
+            <Card className="connection-card" fluid padded="very">
+                <Card.Content>
+                    <Header as="h3" className={ " " }> Advance Configurations </Header>
```

Comment:
Shall we remove the space on either side of the header here? Also, can we add all the texts to the locale, please?

Reply:
Locale we will add later.